src/main/java/core/basesyntax/Colori.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public enum Colori {
```

Comment:
Enum name 'Colori' should be renamed to 'Color'. In Java, it's a convention to use singular nouns for class and enum names.